django_ses_plus/models.py

```diff
@@ -32,12 +33,32 @@ def __str__(self):
         return f"{self.subject} to {self.to_email}"
 
 
+class SentEmailAttachment(models.Model):
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+    sent_email = models.ForeignKey(to="django_ses_plus.SentEmail", related_name="attachments", on_delete=models.CASCADE)
+    name = models.CharField(max_length=255)
+    content = models.BinaryField()
+    type = models.CharField(max_length=255)
```

Comment:
content_mimetype or mimetype is more suitable name. Because this is send as a paramater to email.`attach_alternative` method which accepts mimetype. Please also add with sample values like "text/html, application/pdf, image/png..."